src/components/organisms/Chatbox/Chatbox.tsx

```diff
@@ -69,7 +73,7 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
         }
       : undefined
   );
-  const chats = useSelector(venueChatsSelector);
+  const chats = useVenueChats();
   const privateChats = useSelector(privateChatsSelector);
```

Comment:
Should we be creating a custom hook for private chats (eg. `usePrivateChats` + `usePrivateChatsConnect`) in a similar way as `useVenueChats`? Maybe not in this PR necessarily.

Reply:
Agreed, I think we should move all selector\connect logic to the hooks. Not in this PR though

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/109